xbmc/pvr/PVRManager.cpp

```diff
@@ -1647,18 +1650,21 @@ std::string CPVRManager::GetPlayingTVGroupName()
   return IsStarted() && m_guiInfo ? m_guiInfo->GetPlayingTVGroup() : "";
 }
 
-void CPVRManager::UpdateLastWatched(CPVRChannel &channel)
+void CPVRManager::UpdateLastWatched(const CPVRChannelPtr &channel)
 {
+  if (!channel)
```

Comment:
@opdenkamp Of course you know that. ;-) And now I even understood what you ment exactly. Point taken.